python/mxnet/symbol.py

```diff
@@ -898,17 +981,19 @@ def save(self, fname):
         """Saves symbol to a file.
 
         You can also use pickle to do the job if you only work on python.
-        The advantage of load/save is the file is language agnostic.
-        This means the file saved using save can be loaded by other language binding of mxnet.
-        You also get the benefit being able to directly load/save from cloud storage(S3, HDFS)
+        The advantage of `load`/`save` functions is that the file is language agnostic.
+        This means that the file saved using `save` function can be loaded by other
```

Comment:
Suggest changing this to:
This means the model saved by one language binding can be loaded by a different language binding of MXNet.